components/notebook-controller/config/manager/params.env

```diff
@@ -1,5 +1,8 @@
 USE_ISTIO=true
+USE_AMBIENT_MODE=false
 ISTIO_GATEWAY=kubeflow/kubeflow-gateway
+GATEWAY_NAME=kubeflow-gateway
+GATEWAY_NAMESPACE=kubeflow
 ISTIO_HOST=*
```

Comment:
To keep things clear, renaming those to `K8S_GATEWAY_NAME` AND `K8S_GATEWAY_NAMESPACE` would help.

Reply:
(resolved)